src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -4707,8 +4707,11 @@ internal static ImmutableArray<MethodSymbol> GetReducedAndFilteredMethodGroupSym
             ref LookupResultKind resultKind,
             CSharpCompilation compilation)
         {
-            Debug.Assert(singleResult.Kind != LookupResultKind.Empty);
```

Comment:
>Debug.Assert(singleResult.Kind != LookupResultKind.Empty); [](http://example.com/codeflow?start=12&length=58)

Is it worth keeping this assert, i.e. moving it below the ```if (singleResult.Symbol is null)``` check?

Reply:
I thought that if we had a non-null Symbol, then we can be sure we don't have Kind == LookupResultKind.Empty, and there isn't a need to check. I could be wrong about that, though.